snapshot-service-impl/src/main/java/org/duracloud/snapshot/service/impl/RestoreManagerImpl.java

```diff
@@ -114,6 +114,14 @@ public void setStoreClientHelper(StoreClientHelper storeClientHelper) {
         this.storeClientHelper = storeClientHelper;
     }
 
+    /**
+     *
+     * @param eventLog the event log to set
+     */
+    public void setEventLog(EventLog eventLog) {
```

Comment:
Is this added only to support a test? Is there really a need to capture events in the test context. In any case if these are only for tests they should be protected scope and the javadoc should indicate this as well (to make clear why the method isn't in the RestoreManager interface).